all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -28,6 +28,32 @@ parameters:
   name: JAEGER_ZIPKIN_SERVICE_NAME
   required: true
   value: zipkin
+- description: Limit the number of traces stored in-memory, see https://www.jaegertracing.io/docs/latest/deployment/#memory
+  displayName: Max Traces
+  name: MAX_TRACES
+  required: true
+  value: "1000"
```

Comment:
This should be way higher. Perhaps something like 50k?

Reply:
Sounds reasonable to me. Updated.